wandb/sdk/wandb_run.py

```diff
@@ -2194,6 +2203,13 @@ def _create_image_job(
         artifact = self.log_artifact(job_artifact)
         return artifact
 
+    def log_job_artifact(self, docker_image_name: str) -> Artifact:
```

Comment:
since this function will only create an image job, lets put "image" in the function name. Also prefix this with an underscore so users don't try to call it themselves